upcoming/ssh-server-in-go.md

```diff
@@ -0,0 +1,240 @@
++++
+author = ["Jiahua Chen"]
+date = "2015-12-18T04:07:33-05:00"
+title = "Writing SSH server in Go"
+series = ["Advent 2015"]
++++
+
+When I'm working on [Gogs](https://gogs.io) project, there is a need of builtin
```

Comment:
Considering adding _the_ to _When I'm working on [Gogs](https://gogs.io) project_ so it reads _When I'm working on **the** [Gogs](https://gogs.io) project_